django_socio_grpc/tests/test_run_server.py

```diff
@@ -0,0 +1,243 @@
+from unittest import mock
+
+from django.conf import settings
+from django.core.management import call_command
+from django.test import TestCase, override_settings
+
+from django_socio_grpc.settings import grpc_settings
+from django_socio_grpc.tests.utils import patch_open
+
+
+class FakeAsyncAioServer:
```

Comment:
You can use MagicMock with spec to have a true mocked object

Like so `mock.MagicMock(spec=grpc.aio._server.Server)`